src/commons/api/rrdp.rs

```diff
@@ -99,13 +98,13 @@ impl PublishElement {
         &self.uri
     }
     pub fn size(&self) -> usize {
-        self.base64.size()
+        self.base64.to_bytes().len()
```

Comment:
This operation has become much more costly as it now does a base64 decode to get the bytes, does that matter?

Reply:
Now using better size_approx from rpki-rs